pyproject.toml

```diff
@@ -5,7 +5,7 @@ requires = [
     "setuptools>=42",
     "wheel",
     "pybind11>=2.7.0",
-    "pkgconfig>=0.29.2",
+    "pkgconfig>=l.5.0",
```

Comment:
That looks suspiciously like an "l" not a "1", is this really correct?

Reply:
Happy to approve and merge mod this one typo!

Reply:
ah sorry!